src/v2/view-builder/internals/BaseOktaVerifyChallengeView.js

```diff
@@ -36,6 +42,7 @@ const Body = BaseFormWithPolling.extend({
     this.listenTo(this.model, 'error', this.onPollingFail);
     this.doChallenge();
     this.startPolling();
+    setTimeout(this.stopPolling.bind(this), DEFAULT_POLLING_TIMEOUT);
```

Comment:
Don't we need to cancel this stopPolling function if polling succeeds?

Reply:
I do not think so. If polling is succeded, then the user is redirected and this timer will be cleaned

Reply:
If polling succeeds, then this remediation will be satisfied and widget will proceed to next step in flow - which may or may not be redirect. There should be logic to cleanup this timeout.

Reply:
added cleanup